### DISCUSSION TITLE
第2节课作业(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#99] [作业]
第2节课作业(3班)👇

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/06/mmtutorial-2-assignment/

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/676395232

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/sx7igbx9x9gk7b0z?singleDoc# 《浦语大模型趣味 Demo作业》

**Comment by zzmbhm** [comment]
https://docs.qq.com/doc/DZlJvc3NodExXemti

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/mQgNw

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/676430492

**Comment by Gty-2002** [comment]
https://npumgk6rgc.feishu.cn/docx/RUgydCJz4oxn0VxPBpKcbWcCnKd?from=from_copylink

**Reply by Gty-2002** [reply]
作业在笔记下方！

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135439000?spm=1001.2014.3001.5502

**Comment by semple030228** [comment]
https://blog.csdn.net/m0_49289284/article/details/135439862

**Comment by xiexiaoshinick** [comment]
https://zhuanlan.zhihu.com/p/676521324 作业在文章后半部分

**Comment by SuperD0122** [comment]
https://blog.csdn.net/m0_43404473/article/details/135443505?spm=1001.2014.3001.5501 作业在文章后半部分

**Comment by strawberrylazy** [comment]
https://github.com/strawberrylazy/InterLM_Lesson/blob/main/lesson_2/%E7%AC%AC%E4%BA%8C%E8%8A%82.md

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/676562047

**Comment by 443752009** [comment]
http://t.csdnimg.cn/snauT

**Comment by 443752009** [comment]
http://t.csdnimg.cn/pAa5e

**Comment by Sampson425** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135449163

**Reply by Sampson425** [reply]
https://mp.csdn.net/m0_49908454/article/details/135449163

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135480044

**Comment by Zhihao-z** [comment]
https://github.com/MAMOB/InternLM-Camp-2024/tree/main/Assignment-1
